webpack.config.js

```diff
@@ -512,6 +516,12 @@ if (isDev || isTest) {
 
 }
 
+// reduce logging to optize netlify build
+if (process.env.IS_NETLIFY === "true") {
+    stats: 'errors-only'
```

Comment:
@alisman shouldn't this be `config['stats'] = 'errors-only';`

Also i tend to prefer env variables like `BUILD_REPORT_ERRORS_ONLY=true`, instead of referring to the CI platform. That way it's more clear what the variable does and we could use it on cirlceci or locally as well for instance. But not a big deal

Reply:
ooh. yeah